docs/6.x/changelog.md

```diff
@@ -328,6 +328,14 @@ LTS starts with `3.51.0` with minor backwards compatible changes added over time
 
 ## 5.x Releases
 
+
+### 5.5.24 LTS (October 15th, 2019)
+
+#### Improvements
+
+* Add ability to override peer connect timeout when joining a cluster ([#777](https://github.com/gravitational/gravity/pull/777)).
```

Comment:
Should this ability be added somewhere to the docs? Maybe FAQ or install section.